dev/breeze/doc/05_test_commands.rst

```diff
@@ -218,6 +218,29 @@ Here is the detailed set of options for the ``breeze testing providers-integrati
   :alt: Breeze testing providers-integration-tests
 
 
+Running OpenAPI Client tests
+...................................
+
+To run OpenAPI Client tests, you need to have airflow python client packaged in dist folder.
+To package the client, clone the airflow-python-client repository and run the following command:
+
+.. code-block:: bash
+
+   breeze release-management prepare-python-client --package-format both
```

Comment:
Nice!

Reply:
One more comment though. I think it would be great to add a separate "python_client_tests.rst"  in https://github.com/apache/airflow/tree/main/contributing-docs/testing (and add a link to it in https://github.com/apache/airflow/blob/main/contributing-docs/09_testing.rst . Currently the tests are not described in "contributing docs" and they should be there.

Reply:
make sense i have updated now :)